packages/react/src/components/MultiSelect/MultiSelect.stories.js

```diff
@@ -69,7 +69,178 @@ export const Default = (args) => {
   );
 };
 
-export const WithInitialSelectedItems = (args) => {
+Playground.argTypes = {
+  // align: {
+  //   options: ['start', 'end'],
+  //   control: { type: 'select' },
+  // },
+  // children: {
+  //   control: false,
+  // },
+  // className: {
+  //   control: false,
+  // },
+  // size: {
+  //   options: ['sm', 'md', 'lg'],
+  //   control: { type: 'select' },
+  // },
+  selectionFeedback: {
+    options: ['top', 'fixed', 'top-after-reopen'],
+    control: { type: 'select' },
+  },
+  size: {
+    options: ['sm', 'md', 'lg'],
+    control: { type: 'select' },
+    defaultValue: 'md',
+  },
+  direction: {
+    options: ['top', 'bottom'],
+    control: { type: 'radio' },
+  },
+  type: {
+    options: ['inline', 'default'],
+    control: { type: 'radio' },
+    defaultValue: 'default',
+  },
+  downshiftProps: {
+    control: false,
+  },
```

Comment:
This one and the others below with the same config can be moved up into the component level argTypes and totally removed from the table.